source/_integrations/fan.markdown

```diff
@@ -53,7 +53,7 @@ automation:
 
 ### Service `fan.set_preset_mode`
 
-Sets a preset mode for fan device.
+Sets a preset mode for fan device. Available preset modes are defined by the integration that supplies the fan entity to Home Assistant. For example, the ESPHome [Speed Fan](https://esphome.io/components/fan/speed.html) component provides by default three available presets: `Low`, `Medium` and `High`. 
```

Comment:
```suggestion
Sets a preset mode for the fan device. Available preset modes are defined by the integration that supplies the fan entity to Home Assistant. For example, the ESPHome [Speed Fan](https://esphome.io/components/fan/speed.html) component provides by default three available presets: `Low`, `Medium` and `High`.
```

Reply:
PS: I understood ESPHome fan presets are broken... so... 🤷

Reply:
... that part was my fault :)